what-is-software-development/what-is-software.md

```diff
@@ -0,0 +1,190 @@
+
+- use light and proximity sensors to check if two objects are close to each other
+- be a Twitter bot that posts a poem every day
+- process large amounts of data, like the bus schedule for the past 40 years
+- connect to different devices, through Bluetooth
+- do things on certain events, like email me whenever a meeting on my calendar comes up
+
+### Oh, the Things You Can Build!
+
+Here are some categories of different software to spend a career and a half exploring:
+
+- Web apps
+- Phone/tablet apps
+- Desktop apps
+- Simulation software
+- System architecture and software
+- Artificial Intelligence software
+- Hardware programming
+- Other programming languages
+
+## Software Has Impact
+
+Software has impact and technology is political. Software affects the world in positive and negative ways. Software can build community, and software can influence people.
+
+- What is the context of this software?
+- What kind of communication is this facilitating?
+- What benefits do the users get?
+- What is being amplified?
+- What are the resources needed for this software?
+- How does this software impact the users?
+- How does this software impact the developers?
+- Who can't access the software?
+- Who can't navigate the software?
+
+As of 2020, there are no absolute answers to how to build "perfect" software. However, the tech industry is attempting to have these conversations. Being informed on how software has impact on people and the world is crucial to advance these conversations.
+
+### !callout-info
+
+## Code of Conduct
+
+It's valuable to know that a lot of software and developer communities have adopted a code of conduct, or an agreement and commitment to community health. A popular code of conduct for software is based on [Contributor Covenant](https://www.contributor-covenant.org/). Python's code of conduct is also [available online](https://www.python.org/psf/conduct/).
+
+### !end-callout
+
+## Hello, World!
+
+A "Hello, World!" program is the name of a program whose responsibility is to print `Hello, World!`.
+
+The program is small and simple on purpose. "Hello, World!" programs are a common way to introduce a programming language. A "Hello, World!" program should prove to the programmer that the program runs, and they understand some basic syntax.
+
+It's tradition to write a "Hello, World!" program when starting to learn new language!
+
+In Python, we could open up a Python REPL on our local machine, and execute
+
+```python
+print("Hello, World!")
+```
+
+### !callout-info
+
+## You are the Local Machine
+
+What does "local machine" mean? The phrase "local machine" will be used to mean your own very computer that you're writing code on. "Local" refers to a computer close to you, the user. We say "machine" to not assume that the users of code are laptops or desktop computers!
+
+### !end-callout
+
+## Check for Understanding
+
+<!-- Question 1 -->
```

Comment:
@tildeee these questions LGTM - did you want more before closing #36?

(I can add some more, if you'd like me to.)

Reply:
Sounds like we're good!! Thanks :D !!